CONTRIBUTING.md

```diff
@@ -26,23 +23,24 @@ The easiest way of contributing is forking the repository and submitting a pull
 Please follow these steps to have your contribution considered by the maintainers:
 
 ### For Basic files:
+
 1. Contribute with source file ending in .bas extension (not binary). Filename must be without space and lowercase
-2. Bas file must be in ascii UTF-8 with uppercase letter 
-3. If needed renumber them using the renumber tool (see tools directory)
-4. if subroutine, provide at least one usage example
+1. Bas file must be in ASCII UTF-8 with uppercase letters 
+1. If needed renumber them using the renumber tool (see tools directory)
```

Comment:
Why not leave the numbers? The point of MarkDown is that it is readable even in plain text.

Reply:
The idea was that it would be easier to add or remove list items, but I changed it back to 1-5 instead of just 1 in order that the file should be more readable without a MarkDown converter.